.github/workflows/publish-image.yml

```diff
@@ -42,13 +44,15 @@ jobs:
         with:
           context: horaemeta
           push: true
-          tags: apache/horaemeta-server:latest,apache/horaemeta-server:${{ inputs.version }}
```

Comment:
This will skip `latest` in all tags, what I want is to skip latest only when tags contains `rc`, for tags like `v.2.1.0`, we should include latest tag.